pages/writingguide.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Writing guide
+description: 'When writing about accessibility, here are some things to keep in mind'
+permalink: /writingguide/
+page_title: Accessibility Writing Guide
+---
+
+### The Law
+
+Section 508 is part of the 'Rehabilitation Act of 1973.' The law was amended in 1998 to add 508 which requires that government agencies provide equal access to infromation to disabled employees.
+
+The correct way to refer to the law is: 
+
+* Section 508 of the Rehabilitation Act
+* Section 508 of the Rehabilitation Act of 1973, as amended (29 U.S.C. § 794 (d))
+* Section 508
```

Comment:
Can you rewrite this in order from most to least specific, or from least to most specific?